src/language/dafnyInstallation.ts

```diff
@@ -196,6 +228,13 @@ export class DafnyInstaller {
     );
   }
 
+  private getCustomInstallationPath(typeArch: string): Uri {
+    return Utils.joinPath(
+      this.getInstallationPath(),
+      ...[ 'custom', typeArch ]
+    );
```

Comment:
This should be sufficient:
```suggestion
    return Utils.joinPath(this.getInstallationPath(), 'custom', typeArch);
```